packages/blade/.storybook/react-native/Storybook.tsx

```diff
@@ -8,6 +8,7 @@ import { name as appName } from '../../app.json';
 
 const App = (): React.ReactElement => {
   const Storybook = getStorybookUI({
+    shouldPersistSelection: true,
```

Comment:
Improve RN DX by 10x. 🚀 no more storybook changing the story to first one on hot refresh.

Reply:
10x seems less. Should be 100x 😆